build/Dockerfile

```diff
@@ -1,5 +1,7 @@
+ARG BUILD_IMAGE=golang:1.17.3-alpine3.14
```

Comment:
This should be 1.19 right? I'd suggest to use a sha-linked image reference like: `
ARG GO_IMAGE=golang:1.19.3@sha256:10e3c0f39f8e237baa5b66c5295c578cac42a99536cc9333d8505324a82407d9`

and also to use a distroless image: `FROM gcr.io/distroless/static-debian11@sha256:5759d194607e472ff80fff5833442d3991dd89b219c96552837a2c8f74058617 as runner` as the runner.

Source: https://github.com/mattermost/perseus/blob/main/build/Dockerfile